inspection/background.js

```diff
@@ -12,24 +12,24 @@ const STATUS_PAUSE = 'paused';
 const STATUS_STOP = 'stop';
 const STATUS_CLEAR = 'clear';
 
-var inspectingTab = INSPECT_TAB;
+var inspectingTab;
```

Comment:
still not convinced that `inspectTab` should be global variable. Can't it be localized within `stop()`, `updateBadge()`?

Reply:
Removed inspectTab  and managed locally @mikeliucc